main7.py

```diff
@@ -119,6 +121,7 @@ def test_3_different_value_for_different_ids(self):
         text = self.send_request()
         if text :
             json_text = json.loads(text)
+            # you are using id1 in this function, while you used id variable for the same purpose in another function.

```

Comment:
In this function I need to compare 2 different id's , and that is why I use id1 and id2 variables